prow/plugins/jira-linker/jira-linker_test.go

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
nit: This copyright header is probably not appropriate if we're not upstreaming this, and I don't think we'll be upstreaming this plugin.

Reply:
This is actually added by one of their linters which is useful to us for upstreaming changes; I'm happy to remove it here and stop it being ran by the `update-all.sh` (akin to `pr-check-fix`) script on the `improbable` branch though?